### PR TITLE
Warning if repeated test suite's name

### PR DESCRIPTION
**Pull request by rgonzalezfluendo**
Whenever I created a sub-test suite from another I forget to change the name and realize about the mistake is not easy. This warning can help.

### CODE REVIEW

File: fluster/main.py [hunk 1]
```diff
@@ -37,7 +37,6 @@ class Main:
     # pylint: disable=broad-except
 
     def __init__(self):
-        self.test_suites_dir = TEST_SUITES_DIR
```
Comment (pamarcos) [inline fluster/main.py:40]: Why did you remove this line?

Comment (rgonzalezfluendo) [reply fluster/main.py:40]: It's not needed. It should have been deleted in commit https://github.com/fluendo/fluster/commit/9a1a8e7027c969dad8a5983225d9d2fd8e3f44c5

Comment (pamarcos) [reply fluster/main.py:40]: I see. If you had done a separated commit specifying that, it would have been a no-brainer

**Review by pamarcos — APPROVED** [review]
